### ISSUE TITLE
Osteochondral lesion of talus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11205]
**Preferred term label:**
Osteochondral lesion of talus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Ankle x-ray, CT, or MRI showing articular cartilage damage of the talus bone as suggested by subtle lucency or bone fragmentation, subchondral cyst, subchondral cancellous bone, displaced or separated osteochondral fragment, or high fluid signal in cartilage defect; osteochondral defect

https://uts.nlm.nih.gov/uts/umls/concept/C2015237

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteochondral lesion of the talus (OLT)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001296
- label: Osteochondral lesion of talus
- parents: [Abnormal talus morphology (HP:0008365)](https://hpo.jax.org/app/browse/term/HP:0008365)
- pmid: PMID:34386361|PMID:30498800
- def: Osteochondral lesion of the talus (OLT) is an injury involving the articular cartilage and subchondral bone, defined as any damage involving both articular cartilage and subchondral bone of the talar dome. OLT can cause ankle joint pain and can sometimes lead to instability and manifest as an area of abnormal, damaged cartilage and bone on the top of the talus bone (the lower bone of the ankle joint).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11205
- synonyms:
